samples/automanage/README.md

```diff
@@ -0,0 +1,89 @@
+---
+page_type: sample
+languages:
+- python
+products:
+- azure
+description: "These code samples will show you how to manage Automanage using Azure SDK for Python."
+urlFragment: Automanage
+---
+
+# Getting started - Managing Automanage using Azure Python SDK
+
+These code samples will show you how to manage Automanage using Azure SDK for Python.
+
+## Features
+
+This project framework provides examples for the following services:
+
+### Automanage
+* [] Using the Azure SDK for Python - Automanage Management Library [azure-mgmt-automanage](https://pypi.org/project/azure-mgmt-automanage/) for the [Automanage API](https://docs.microsoft.com/en-us/rest/api/automanage/)
+
+`pip install azure-mgmt-automanage`
+
+## Getting Started
+
+### Prerequisites
+
+1. Before we run the samples, we need to make sure we have setup the credentials. Follow the instructions in [register a new application using Azure portal](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal) to obtain `subscription id`,`client id`,`client secret`, and `application id`
+
+2. Store your credentials an environment variables.
+For example, in Linux-based OS, you can do
+```bash
+export AZURE_TENANT_ID="xxx"
+export AZURE_CLIENT_ID="xxx"
+export AZURE_CLIENT_SECRET="xxx"
+export SUBSCRIPTION_ID="xxx"
+```
+
+### Installation
+
+1.  If you don't already have it, [install Python](https://www.python.org/downloads/).
+
+    This sample (and the SDK) is compatible with Python 3.7+.
+
+2.  General recommendation for Python development is to use a Virtual Environment.
+    For more information, see https://docs.python.org/3/tutorial/venv.html
+
+    Install and initialize the virtual environment with the "venv" module on Python 3 (you must install [virtualenv](https://pypi.python.org/pypi/virtualenv) for Python 2.7):
```

Comment:
This sentence can be ignored, because it has been proposed to use python3.7+ above.